robottelo/ui/usergroup.py

```diff
@@ -59,31 +59,25 @@ def update(self, old_name, new_name=None,
             roles = []
         if new_roles is None:
             new_roles = []
-        element = self.search(old_name)
-
-        if element:
-            element.click()
-            self.wait_for_ajax()
-            if new_name:
-                if self.wait_until_element(locators['usergroups.name']):
-                    self.field_update('usergroups.name', new_name)
-            self.configure_entity(
-                users, FILTER['usergroup_user'], new_entity_list=new_users)
-            if roles or new_roles:
-                self.click(tab_locators['usergroups.tab_roles'])
-                if "admin" in roles or "admin" in new_roles:
-                    self.click(locators['usergroups.admin'])
-                else:
-                    self.configure_entity(
-                        entity_list=roles,
-                        new_entity_list=new_roles,
-                        filter_key=FILTER['usergroup_role'],
-                        entity_select=entity_select,
-                    )
-            if refresh_extusrgp:
-                self.click(tab_locators['usergroups.tab_external'])
-                self.click(locators['ldapserver.refresh'])
+        self.click(self.search(old_name))
+        if new_name:
+            if self.wait_until_element(locators['usergroups.name']):
+                self.field_update('usergroups.name', new_name)
+        self.configure_entity(
+            users, FILTER['usergroup_user'], new_entity_list=new_users)
+        if roles or new_roles:
+            self.click(tab_locators['usergroups.tab_roles'])
+            if "admin" in roles or "admin" in new_roles:
```

Comment:
By reading this code I can assume that if admin is passed in roles or new_roles then no other role will be added. I am not familiar with the feature by it makes me think that if admin is specified then no other role or new_role is not needed to be set.